test/resources/billing.test.js

```diff
@@ -2,39 +2,72 @@
 import { expect } from 'chai';
 
 import EasyPost from '../../src/easypost';
-import * as setupPolly from '../helpers/setup_polly';
+import {
+  MockMiddleware,
+  MockRequest,
+  MockRequestMatchRule,
+  MockRequestResponseInfo,
+} from '../helpers/mocking';
+
+const middleware = (request) => {
+  return new MockMiddleware(request, [
+    new MockRequest(
+      new MockRequestMatchRule('POST', 'v2\\/bank_accounts\\/\\S*\\/charges$'),
+      new MockRequestResponseInfo(200, {}),
+    ),
+    new MockRequest(
+      new MockRequestMatchRule('POST', 'v2\\/credit_cards\\/\\S*\\/charges$'),
+      new MockRequestResponseInfo(200, {}),
+    ),
+    new MockRequest(
+      new MockRequestMatchRule('DELETE', 'v2\\/bank_accounts\\/\\S*$'),
+      new MockRequestResponseInfo(200, {}),
+    ),
+    new MockRequest(
+      new MockRequestMatchRule('DELETE', 'v2\\/credit_cards\\/\\S*$'),
+      new MockRequestResponseInfo(200, {}),
+    ),
+    new MockRequest(
+      new MockRequestMatchRule('GET', 'v2\\/payment_methods$'),
+      new MockRequestResponseInfo(200, {
+        id: 'summary_123',
+        primary_payment_method: {
+          id: 'card_123',
+          last4: '1234',
+        },
+        secondary_payment_method: {
+          id: 'bank_123',
+          bank_name: 'Mock Bank',
+        },
+      }),
+    ),
+  ]);
+};
 
 describe('Billing Resource', function () {
-  setupPolly.startPolly();
 
   before(function () {
-    this.easypost = new EasyPost(process.env.EASYPOST_PROD_API_KEY);
+    this.easypost = new EasyPost(process.env.EASYPOST_TEST_API_KEY, {
+      requestMiddleware: middleware,
+    });
   });
 
   beforeEach(function () {
-    const { server } = this.polly;
-    setupPolly.setupCassette(server);
   });
```

Comment:
This function can be removed

Reply:
Keeping just in case we need to re-add something to it in the future (if we add new unit tests that need some kind of pre-test functionality).

Reply:
We won't forget the syntax because there are dozens of other examples in this repo of how to do it and if we need it in the future, we would add it in the future. This is dead code that multiple people asked to have removed during code review.